README.md

```diff
@@ -16,27 +18,25 @@ Follow the OS-specific instructions to install the [Azure Functions Core Tools](
 
 Install the prerequisites for your desired language:
 
-* [JavaScript](#javascript)
-* [TypeScript](#javascript)
-* [C#](#c)
+* [JavaScript](#node)
+* [TypeScript](#node)
+* [C#](#net)
 * [Java](#java)
 * [Python (Preview)](#python)
 
-> **NOTE**: You may change your `azureFunctions.projectLanguage` user setting to multiple other 'preview' languages not listed above. This allows you to create a project/function in that language, but we do not yet support local debugging for these languages.
-
 ## Features
 
-* Create new Function projects
-* Create new Functions from a template
-* Debug Function projects locally
-* Deploy to Azure Function Apps
-* View, create, delete, start, stop, and restart Azure Function Apps
+* Create new function projects
```

Comment:
FWIW, if you look at https://azure.microsoft.com/en-us/services/functions/, they do capitalize Function and App when referring to Azure Function Apps or Azure Functions or even Functions as a product, while keeping it lowercase for stuff like "creating your first function".